database/kwdb.xml

```diff
@@ -10161,7 +10161,7 @@ Returns the type constant for the value in json indicated by specifiers
   <function name="llReturnObjectsByID" type="integer" grid="sl">
     <param type="list" name="objects"/>
     <description lang="en">
-Returns an integer that is the number of objects successfully returned to their owners or an ERR_* flag.llReturnObjectsByOwner
+Returns an integer that is the number of objects successfully returned to their owners or an ERR_* flag.llReturnObjectsByOwner.
```

Comment:
I don't think `llReturnObjectsByOwner` should appear there in the first place.

Reply:
Also, the description talks about the return value, but not about what the function does. Something along these lines would work better, I think:

> Returns to their owners the objects with the ids given in a list. The return value is the number of objects successfully returned, or an ERR_* flag.

And something similar for `llReturnObjectsByOwner`.